Library/Homebrew/formulary.rb

```diff
@@ -155,6 +157,34 @@ def self.class_s(name)
     class_name
   end
 
+  def self.formula_from_json(name, json)
+    hash = JSON.parse json
```

Comment:
May want to test what happens if this isn't parseable and have a more graceful failure.